cirq-superstaq/cirq_superstaq/ops/qubit_gates.py

```diff
@@ -389,7 +389,17 @@ def _circuit_diagram_info_(self, args: cirq.CircuitDiagramInfoArgs) -> Tuple[str
         return ("|",) * self.num_qubits()
 
 
-def barrier(*qubits: cirq.Qid) -> cirq.Operation:  # pylint: disable=missing-function-docstring
+def barrier(*qubits: cirq.Qid) -> cirq.Operation:
+    """Cirq equivalent of
+    https://qiskit.org/documentation/stubs/qiskit.circuit.library.Barrier.html.
+    See also https://github.com/quantumlib/Cirq/issues/2642.
+
+    Args:
+        *qubits: The qubits that the barrier will cover.
+
+    Returns:
+        A barrier `cirq.Operation` on the passed in qubits.
```

Comment:
nit: should be either hyphenated
```suggestion
        A barrier `cirq.Operation` on the passed-in qubits.
```
or replaced with a different word e.g.
```suggestion
        A barrier `cirq.Operation` on the provided qubits.
```